src/components/AccordionSection/index.js

```diff
@@ -0,0 +1,176 @@
+/* eslint-disable react/prop-types */
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import classnames from 'classnames';
+import RenderIf from './../RenderIf';
+import { uniqueId } from './../../libs/utils';
+import { Consumer } from './../Accordion/context';
+import ButtonIcon from './../ButtonIcon';
+import RightArrow from './rightArrow';
+import isInArray from './check-is-in-array';
+import './styles.css';
+
+class AccordionItem extends Component {
+    constructor(props) {
+        super(props);
+        this.accordionDetailsId = uniqueId('accordion-details');
+        this.name = uniqueId('accordion-section');
+        this.handleSelect = this.handleSelect.bind(this);
+    }
+
+    getContainerClassNames() {
+        const { className, disabled } = this.props;
+        return classnames('rainbow-accordion_container', {
```

Comment:
You must use rainbow-accordion-section_container since this component is the AccordionSection and not the Accordion. This apply for all classNames in this component.